### PR TITLE
Feat/ratik/zmq

### PR DESCRIPTION
**Pull request by brooke-k**
- Moved all contents of src/redux/ to src/redux-features/
- Renamed NetworkDispatch.tsx to NetworkDispatchStore.tsx for consistency
- Renamed AuxillarySensors.tsx to AuxillarySensorStore.tsx for consistency
- Moved selectors from NetworkDispatchStore.tsx to src/app/reducersindex.tsx
- Moved selectors from AuxillarySensorsStore.tsx to src/app/reducersindex.tsx
- Added 'jszmq' as a yarn package, needed for NetworkDispatch
- Resolved merge conflicts with main to allow for rebasing

### CODE REVIEW

File: src/Models/Interfaces.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    controller: ControllerTypes;
+    data: IBatteryControllerReceivable | IMotorControllerReceivable | IClimateControllerReceivable;
+}
+
+/**
+ * Message format of transmitted socket messages
+ */
+export interface ISocketMessageTransmittable {
+    controller: ControllerTypes;
+    data: any;
```
Comment (RatikKapoor) [inline src/Models/Interfaces.ts:17]: We should eventually refactor this into a generic type with different messages

**Review by RatikKapoor — APPROVED** [review]
Looks good to me! Definitely a great start to get systems and the frontend all linked together!

**Review by muhas1 — APPROVED** [review]
LGTM!

File: src/Models/Interfaces.ts (hunk 1 above)
Comment (brooke-k) [reply src/Models/Interfaces.ts:17]: Fully agreed - I can go in and fix it in a bit. At the moment it works but it definitely needs to be cleaned up.

**Pull request merged**